### ISSUE TITLE
New Instructor instructions added to "tickets" e-mail

### ISSUE THREAD

**Issue by jrteuton** [carpentries/handbook#356]
I would suggest in the future the "ticket" email have a set of links/clear instructions on it for steps that need to be taken before a workshop.

For example to confirm a correct link/sign in for online courses or directions to a physical location and any steps to gain access, if a facility requires sign in or other access control be completed. 

I would also suggest a list/links to "homework" Setup/download/reading that should be completed before the event begins. This might make things go smoother on the day of, especially for online courses.

**Comment by maneesha** [comment]
Thanks for sharing your suggestion, @jrteuton .  Can you tell us specifically which email you are referring to?  Linking to it in the handbook would be ideal.

**Comment by jrteuton** [comment]
the "ticket" e-mail you get after registering for a class, in this case it was the instructors online course.  
    On Thursday, January 17, 2019, 3:39:13 PM PST, maneesha sane <notifications@github.com> wrote:  
 
 
Thanks for sharing your suggestion, @jrteuton . Can you tell us specifically which email you are referring to? Linking to it in the handbook would be ideal.

**Comment by maneesha** [comment]
Hi @jrteuton  -- Thanks for your follow up.  Is this the email that was sent directly from Eventbrite or that a Carpentries staff member sent you?

**Comment by jrteuton** [comment]
I believe it was everbrite ... I don't think I received an email from the staff before the event ... I'm pretty sure the survey I filled out where I generating a unique identifier based on my birth city excetera, didn't complete properly. I did it with a cell phone limited signal the weekend before. Don't remember getting the confirmation screen. 
Jeremy 

Sent from Yahoo Mail on Android 
 
  On Fri, Feb 8, 2019 at 9:53 AM, maneesha sane<notifications@github.com> wrote:   
Hi @jrteuton -- Thanks for your follow up. Is this the email that was sent directly from Eventbrite or that a Carpentries staff member sent you?
